src/test/java/no/nav/tag/sykefravarsstatistikk/api/sammenligning/SykefraværprosentRepositoryTest.java

```diff
@@ -55,6 +60,16 @@ public void setUp() {
         assertThat(sql).contains(":" + KVARTAL);
     }
 
+    @Test
+    public void hentSykefraværprosentLand_skal_returnere_tomt_sykefravær_hvis_database_ikke_inneholder_rader() {
+        when(jdbcTemplate.queryForObject(anyString(), any(SqlParameterSource.class), any(RowMapper.class)))
+                .thenThrow(new EmptyResultDataAccessException(0));
+
```

Comment:
Lurer på om vi kunne, i stedet av å _mocke_ det JDBC Template gjør ved _empty resultset_ , bruke direkte repository med In-memory DB. Litt som `DatavarehusRepositoryJdbcTest` gjør